features/step_definitions/lolcommits_steps.rb

```diff
@@ -73,7 +73,7 @@
 end
 
 Then /^I should be (prompted for|presented) "(.*?)"$/ do |_, prompt|
-  assert @stdout.read.to_s.include?(prompt)
+  expect(@stdout.read.to_s).to include(prompt)
```

Comment:
Ah I think I see why.. this is so we can remove the dependency on TestUnit assertions within step definitions?  And use the rspec-expectations available through aruba..

It kinda sucks a bit that rspec leaks in here tho.  I wonder if Aruba can work with MiniTest like assertions somehow.